pkg/bindings/images/diff.go

```diff
@@ -9,7 +9,11 @@ import (
 )
 
 // Diff provides the changes between two container layers
-func Diff(ctx context.Context, nameOrID string) ([]archive.Change, error) {
+func Diff(ctx context.Context, nameOrID string, options *DiffOptions) ([]archive.Change, error) {
+	if options == nil {
```

Comment:
Great idea to allow `nil`!

I think it's a nice compromise between avoiding future breaking changes (if we have the need to at fields to the currently empty struct) and being easy to use: `Diff(a, b, nil)` is nicer than `Diff(a, b, &images.DiffOptions)`.